_posts/blog/2016-10-05-dwh-01.md

```diff
@@ -23,3 +23,5 @@ Given all these different use cases, it was clearly __time for a big data soluti
 In a series of articles we would like to __share our experiences__ with you and invite you on our __journey towards a functional custom data warehouse solution__. See us fail, despair, hope, fail again and maybe finally succeed in our undertaking.
 
 ___Let our journey begin…___
+
+<a href="../dwh-part1_getting_started/">A journey towards a custom data warehouse solution Part 1 : Getting Started</a>
```

Comment:
Please change it to:
A Journey Towards a Custom Data Warehouse Solution - Part 1: Getting Started